examples/nodejs/lambda-examples/cloudwatch-metrics/lambda/handler.ts

```diff
@@ -0,0 +1,86 @@
+import {GetSecretValueCommand, SecretsManagerClient} from '@aws-sdk/client-secrets-manager';
+import {
+  CacheClient,
+  Configurations,
+  DefaultMomentoLoggerFactory,
+  DefaultMomentoLoggerLevel,
+  CredentialProvider,
+} from '@gomomento/sdk';
+import {ExperimentalMetricsLoggingMiddleware} from '@gomomento/sdk/dist/src/config/middleware/experimental-metrics-logging-middleware';
+
+const _secretsClient = new SecretsManagerClient({});
+const _cachedSecrets = new Map<string, string>();
+let _cacheClient: CacheClient | undefined = undefined;
+
+export const handler = async () => {
+  try {
+    const loggerFactory = new DefaultMomentoLoggerFactory(DefaultMomentoLoggerLevel.DEBUG);
+    const logger = loggerFactory.getLogger('CloudWatchMetricsMiddlewaresExample');
+
+    const cacheClient = await getCacheClient(loggerFactory);
+    logger.info('Created a CacheClient configured with metrics middleware');
+
+    logger.info('Issuing 5 minutes of set and get requests to generate data for the dashboard example');
+    for (let i = 0; i < (60 /* seconds */ * 5 /* minutes */); i++) {
```

Comment:
momento api calls should be fast so this should probably not matter, but just calling out that this isnt actually 5 min of gets and sets. Its 300 gets and sets plus a 300 1-second delays. So its 300 x (1 + epsilon) seconds.